orga/app/components/routes/authenticated/campaign/collective-results/tab.js

```diff
@@ -0,0 +1,13 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { htmlSafe } from '@ember/string';
+
+export default class Tab extends Component {
+  @service intl;
+
+  get campaignCollectiveResultLabel() {
+    return htmlSafe(this.intl.t('pages.campaign-collective-results.table.column.competences',
+      { count: this.args.sharedParticipationsCount ? this.args.campaignCollectiveResult.get('campaignCompetenceCollectiveResults').length : '-' },
```

Comment:
`this.args.sharedParticipationsCount` ne prend pas en compte la length de `this.args.campaignCollectiveResult` , du coup , on ne peut pas utiliser plural. 
Ember a suggéré  ( nous a forcé  ) à utiliser le `.get` car il na pas aimé l’accès direct a tout les properties.